src/backend/renderer/gles2/shaders.rs

```diff
@@ -49,9 +57,17 @@ precision mediump float;
 uniform sampler2D tex;
 uniform float alpha;
 varying vec2 v_tex_coords;
+uniform float tint;
 
 void main() {
-    gl_FragColor = vec4(texture2D(tex, v_tex_coords).rgb, 1.0) * alpha;
+    vec4 color;
+
+    color = vec4(texture2D(tex, v_tex_coords).rgb, 1.0) * alpha;
+
+    if (tint == 1.0)
+        color = vec4(0.0, 0.3, 0.0, 0.2) + color * 0.8;
```

Comment:
I think you can write this more clearly with:
`mix(gl_FragColor, vec4(0.0, 1.0, 0.0, 0.8)`